.github/scripts/external_pr_labeller.py

```diff
@@ -0,0 +1,65 @@
+from github import Github, Auth
+import os
+import logging
+
+import argparse
+
+
+def get_arguments() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-r',
+        '--repository-name',
+        type=str,
+        required=True,
+        help='Repository name in the OWNER/REPOSITORY format',
+    )
+    parser.add_argument(
+        '--pr-number', type=int, required=True, help='PR number to label'
+    )
+    return parser.parse_args()
+
+
+def init_logger():
+    LOGLEVEL = os.environ.get('LOGLEVEL', 'INFO').upper()
+    logging.basicConfig(
+        level=LOGLEVEL,
+        format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
+        datefmt='%m-%d-%Y %H:%M:%S',
+    )
+
+
+if __name__ == '__main__':
+
+    init_logger()
+
+    LOGGER = logging.getLogger('labeller')
+    EXTERNAL_PR_LABEL_NAME = 'ExternalPR'
+
+    args = get_arguments()
+    pr_number = args.pr_number
+    repository_name = args.repository_name
+
+    github = Github(auth=Auth.Token(token=os.environ.get('GITHUB_TOKEN')))
+    gh_repo = github.get_repo(full_name_or_id=repository_name)
+
+    pr = gh_repo.get_pull(number=pr_number)
+    
+    LOGGER.info(f'CONTEXT: PR #{pr_number}. USER: {pr.user.login}. ALL PR LABELS: {list(pr.get_labels())}')
+
+    if not gh_repo.has_in_collaborators(pr.user.login):
```

Comment:
- people in organization are supposed to be collaborators? 
- or ask differently, will PR be marked as external if github does not automatically run CI for it? I suppose rules must be the same

Reply:
The people in org are supposed to be collaborators:
* There are 329 Intel members
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/63e814ed-59ad-4a7e-84b8-139469b12941)
+ 3 outside collaborators (one is duplicated from members):
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/001fbbae-f79f-4047-81a2-70f95afec437)

the total number of collaborators is 333 as per the script:
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/421de79d-1b5d-465f-89a3-224c759d6290)
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/c1fc636b-ff7c-4184-8241-b5dc12e99a16)

I believe these are all the people with write access -> for whom the pipelines should start automatically. The `labellel.yml` starts for all PRs as it has the `pull_request_target` trigger.